contrib/babelfishpg_tsql/src/cursor.c

```diff
@@ -1468,6 +1471,42 @@ execute_sp_cursoropen_common(int *stmt_handle, int *cursor_handle, const char *s
 
 	if (prepare)
 	{
+		/*
+		 * This entire block is to parse the statement by antlr and use the resultant
+		 * statement to sent to Postgres cursor execution. This is necessary in some use case,
+		 * for example when we have PostgreSQL reversed keywords in query which is valid in TSQL.
+		 * Antlr parser will add the quotes at necessary places in query so that Postgres engine
+		 * can resolve this query correctly.
+		 */
+		if (stmt)
```

Comment:
Is it possible for white spaced empty stmt to reach here?
If yes, does pltsql_compile_inline handle that safely? Maybe we can add negative test case for this